infra/build/Makefile

```diff
@@ -92,26 +93,9 @@ build-image-developer-tools:
 		--build-arg TINKEY_VERSION=${TINKEY_VERSION} \
 		--build-arg ALPINE_VERSION=${ALPINE_VERSION} \
 		--build-arg MODULE_SWAPPER_VERSION=${MODULE_SWAPPER_VERSION} \
+		--build-arg TFLINT_BP_PLUGIN=${TFLINT_BP_PLUGIN} \
 		-t ${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} "./developer-tools"
-	docker build \
-		--build-arg BASE_IMAGE_VERSION=${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
-		--build-arg PACKER_VERSION=${PACKER_VERSION} \
-		-t ${DOCKER_IMAGE_DEVELOPER_TOOLS_JENKINS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} "./developer-tools-jenkins"
-	docker build . \
-		--build-arg TERRAFORM_VERSION=${TERRAFORM_VERSION} \
-		--build-arg CLOUD_SDK_VERSION=${CLOUD_SDK_VERSION} \
-		--build-arg ALPINE_VERSION=${ALPINE_VERSION} \
-		-t ${DOCKER_IMAGE_DEVELOPER_LIGHT}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} -f "./developer-tools-light/Dockerfile"
-	docker build \
-		--build-arg GOLANGCI_VERSION=${GOLANGCI_VERSION} \
-		--build-arg ASMCLI_VERSION=${ASMCLI_VERSION} \
-		--build-arg GATOR_VERSION=${GATOR_VERSION} \
-		--build-arg OPA_VERSION=${OPA_VERSION} \
-		--build-arg KIND_VERSION=${KIND_VERSION} \
-		--build-arg GCRANE_VERSION=${GCRANE_VERSION} \
-		--build-arg BASE_IMAGE_VERSION=${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
-		-t ${DOCKER_IMAGE_DEVELOPER_KRM}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} "./developer-tools-krm"
```

Comment:
I think there might still be some deps on `developer-tools-krm`?

Reply:
Woops this was totally unintended. I think I was testing the build locally and ended up removing this to make it faster but forgot to restore.